app/templates/buyers/review_brief.html

```diff
@@ -0,0 +1,91 @@
+{% extends "_base_page.html" %}
+{% from "components/tabs/macro.njk" import govukTabs %}
+
+{% block pageTitle %}{{ brief.title }} [DRAFT] - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+          "link": url_for('external.index'),
+          "label": "Digital Marketplace"
+      },
+      {
+          "link": url_for('external.list_opportunities', framework_family=brief.framework.family),
+          "label": "Supplier opportunities"
+      },
+      breadcrumbs
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
```

Comment:
I know the breadcrumbs might not be in this application at the moment but maybe it might be good to get a start on it?

Reply:
I don't think we can use the breadcrumbs partially; the new style runs from the left margin of the page unless the base page layout is changed, and if you change the base page layout old style breadcrumbs get indented weirdly from where they should be.